src/redux/saga/sagaImageDaily.js

```diff
@@ -0,0 +1,27 @@
+import { takeEvery, put, call } from 'redux-saga/effects'
+
+import { imageAPI } from '../../utils/axiosAPI/api'
+import { FETCH_IMAGES_SUCCESS, IS_LOADING, REQUEST_IMAGES } from '../types'
+
+function* fetchImages() {
+  yield put({
+    type: IS_LOADING,
+    payload: {
+      isLoading: true,
+    },
```

Comment:
extract to a function please

Reply:
I understand what you are thinking but I can do this in the future to better understand how the saga works

Reply:
ok